### PR TITLE
Nanayeb34 add new tts script

### PR DESCRIPTION
**Pull request by Nanayeb34**
# Description

Added the optimized TTS competition script

# Reviewers

@wsedor @kojomensahonums @naafidasana

### CODE REVIEW

**Review by kojomensahonums — COMMENTED** [review]
Are you making the models available with the config and model paths you've set? It looks they also point more to a colab environment 'content/'?

**Comment by wsedor** [comment]
@Nanayeb34 this PR has bene out for sometime. If it's no longer needed please close it

**Comment by Nanayeb34** [comment]
@wsedor i think no one has reviewed this PR yet that's why it can't be merged. Could you approve it so i merge it?

File: text-to-speech/General_TTS_inference_API_notebook_version2.ipynb [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+        "id": "xbHL2ZCNNYu3"
+      },
+      "execution_count": 27,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "#Replace comment section below with your auth token\n",
+        "!ngrok config add-authtoken #place your access token here"
```
Comment (wsedor) [inline text-to-speech/General_TTS_inference_API_notebook_version2.ipynb:174]: let's switch this to import the token from the secrets section. Similar to the change in https://github.com/brilla-ai/brilla-ai/commit/9bce29108eafea28da64bdc4b9d755cce7deee91#diff-e237dc95b672c52b333a45cb4f27cba37eecb11242ea598c779f0519b26439a6R24 and https://github.com/brilla-ai/brilla-ai/commit/9bce29108eafea28da64bdc4b9d755cce7deee91#diff-e237dc95b672c52b333a45cb4f27cba37eecb11242ea598c779f0519b26439a6R131

File: text-to-speech/General_TTS_inference_API_notebook_version2.ipynb [hunk 2]
```diff
@@ -172,8 +168,8 @@
     {
       "cell_type": "code",
       "source": [
-        "#Replace comment section below with your auth token\n",
-        "!ngrok config add-authtoken #place your access token here"
+        "token=userdata.get('ngrok_token')\n",
```
Comment (wsedor) [inline text-to-speech/General_TTS_inference_API_notebook_version2.ipynb:171]: can we capitalize the token name and also the key so it's consistent with other notebooks. `"NGROK_TOKEN = userdata.get('NGROK_TOKEN')\n",`

**Review by jojoboateng — APPROVED** [review]

**Pull request merged**